draft-fossati-tls-attestation.md

```diff
@@ -939,14 +939,10 @@ confidential computing properties.
 
 ## IoT Device Onboarding
 
-In this example, an IoT is onboarded to a cloud service provider (or to a
-network operator). In this scenario there is typically no a priori
-relationship between the device and the cloud service provider that 
-will remotely manage the device.
-
-In such scenario, the cloud service provider wants to make sure
-that the device runs the correct version of firmware, has not been 
-rooted, is not emulated or cloned.
+In this scenario, an IoT device is connected to a remote device management entity, which could be a cloud service provider or a network operator.
+We assume that, initially, the remote device management entity does not trust the device.
+
+The device management entity's responsibility is to guarantee that the device is running the intended firmware version, has not been tampered with, and is not being emulated or cloned.
```

Comment:
Don't know if I'm being pedantic, and this is intended to be an editorial fix so feel free to move to a different issue, but why is "firmware version" singled out here? e.g. with a TPM you can measure all the way into userspace

Reply:
> why is "firmware version" singled out here? e.g. with a TPM you can measure all the way into userspace

Right, here the attester is a small thing, so it makes sense to use the term "firmware", I guess.

I am happy to change it to "software".

Reply:
Ah, I see, it's intended strictly as a specific example. I think either option works, leaving it to you to decide.